test/nodetool/test_compact.py

```diff
@@ -37,8 +37,7 @@ def test_nonexistent_keyspace(nodetool):
             ("compact", "non_existent_ks"),
             {"expected_requests": [
                 expected_request("GET", "/storage_service/keyspaces", multiple=expected_request.MULTIPLE,
-                                 response=["system"]),
-                expected_request("POST", "/storage_service/keyspace_compaction/non_existent_ks")]},
+                                 response=["system"])]},
```

Comment:
What's this change?

Reply:
Ok, I didn't understand that **this** commit is what makes the checking stricter. Now I understand (I think...) that the testing just happens in additional cases, which is why this change is needed. Ok.